x/gov/client/rest/rest.go

```diff
@@ -310,7 +310,7 @@ func queryDepositHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext) http.Han
 
 		if len(strProposalID) == 0 {
 			err := errors.New("proposalId required but not specified")
-			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
```

Comment:
this is correct and shouldn't be replaced

Reply:
noted. Reverted.